src/YQMenuBar.cc

```diff
@@ -80,6 +80,8 @@ YQMenuBar::rebuildMenuTree()
 
 	QMenu * menu = QMenuBar::addMenu( fromUTF8( item->label() ));
 	item->setUiItem( menu );
+        menu->menuAction()->setEnabled( item->isEnabled() );
```

Comment:
This line should be
menu->setEnabled( item->isEnabled() );

Reply:
Oops... fixed.

Thanks for spotting this!